tests/test_filters.py

```diff
@@ -146,6 +146,9 @@ def test_clip_box(datasets):
         clp = dataset.clip_box(invert=True, progress_bar=True)
         assert clp is not None
         assert isinstance(clp, pyvista.UnstructuredGrid)
+        clp2 = dataset.clip_box(merge_points=False)
```

Comment:
I wonder if we should try checking that point merging was indeed disabled here. I'm on the fence because we'd mostly be testing VTK, but we'd be testing something non-trivial (interaction of a filter with a locator).

Reply:
I think @basnijholt's original example in #2168 of the two triangles would be an appropriate test. IMO, something simple like this is just before the line of "testing VTK" and within scope

All we'd need to do is `assert clp2.n_points == #`